swal-forms.js

```diff
@@ -46,10 +46,13 @@
     addWayToGetFormValuesInDoneFunction: function(swalArgs) {
       var swalFormInstance = this
       var doneFunction = swalArgs[1]
-      swalArgs[1] = function() {
+      swalArgs[1] = function(isConfirm) {
```

Comment:
it is not used in this code. when you pass an function with no params to sweetAlert, this is called only if you click in the ok button. If this function is declared with one param, this function is called when the swal is canceled too.
![image](https://cloud.githubusercontent.com/assets/8538337/12534302/db124dc2-c230-11e5-8617-1870eb65e729.png)